ymer/src/main/java/com/avanza/ymer/PerformedOperationMetrics.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2015 Avanza Bank AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.avanza.ymer;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class PerformedOperationMetrics implements PerformedOperationMetricsMBean, PerformedOperationsListener {
```

Comment:
I was thinking we should maybe place these new classes in a subpackage, if all new classes are `public` anyway?

The `com.avanza.ymer` package contains so much already 😅

Reply:
Agree, there are, however other MBeans in the base package today and moving them as well to a new package requires opening several of the existing classes (`SpaceMirrorContext` and `DocumentWriteExceptionHandler`)  to `public` so I left it as is. Having one `jmx` package with only one of three MBeans seemed strange ...